src/find/matchers/perm.rs

```diff
@@ -271,8 +280,8 @@ impl PermMatcher {
     }
 
     #[cfg(not(unix))]
-    pub fn new(pattern: &str) -> Result<PermMatcher, Box<Error>> {
-        Err(From("Permission matching is not available on this platform"))
+    pub fn new(_dummy_pattern: &str) -> Result<PermMatcher, Box<Error>> {
```

Comment:
`_` is fine in an _implementation_ that doesn't use the parameter. But when defining a trait where the parameter will be used (even if it isn't in a default implementation) I do prefer a more explicit name.